libyul/backends/wasm/WasmDialect.cpp

```diff
@@ -218,9 +226,18 @@ void WasmDialect::addEthereumExternals()
 			f.returns.emplace_back(YulString(p));
 		// TODO some of them are side effect free.
```

Comment:
Does the new code below take care of this TODO?

Reply:
I don't think so. I think the WASM-SideEffects may need a rehaul.